src/v/cloud_storage/async_manifest_view.cc

```diff
@@ -46,6 +46,15 @@
 #include <system_error>
 #include <variant>
 
+namespace {
+ss::log_level log_level_for_error(cloud_storage::error_outcome err) {
```

Comment:
nit: maybe move to 'types' or 'logger'?

Reply:
Could do, but it's quite specific to the async manifest view.

Reply:
it's OK to keep it here, that's just an idea